gym_gridverse/envs/env.py

```diff
@@ -25,21 +25,21 @@ def __init__(
 
     @abc.abstractmethod
     def set_seed(self, seed: Optional[int] = None):
-        raise NotImplementedError
+        raise AssertionError("Must be implemented by derived class")
```

Comment:
I think this (and others) should just be `assert False`;  same semantic meaning in a more standard form / message

Reply:
Done